src/app/auth/pages/connection-page/connection-page.component.html

```diff
@@ -37,9 +37,12 @@ <h1>{{ title }}</h1>
     </div>
 
     <button type="submit" class="btn btn-info" [disabled]="loginForm.form.invalid">Se connecter</button>
-
   </form>
 
+  <p><a *appPermission="'admin'" id="testButton" class="btn btn-warning" [routerLink]="['/games']">Return to Games</a></p>
+
+  <p><a *myNgIf="loggedIn" id="testButton2" class="btn btn-danger" [routerLink]="['/games']">Games</a></p>
```

Comment:
🤦 ça n'a pas vraiment de sens ... tu fais un ngIf à toi ... le but était d'avoir le même comportement mais sans passer de valeur en input :

```suggestion
  <p><a *afficherSiIdentifie id="testButton2" class="btn btn-danger" [routerLink]="['/games']">Games</a></p>
```